### PR TITLE
Add some periods in cloud controller manager's options

### PR DESCRIPTION
**Pull request by drinktee**
Add some periods in cloud controller manager's options

### CODE REVIEW

File: cmd/cloud-controller-manager/app/options/options.go [hunk 1]
```diff
@@ -67,13 +67,13 @@ func NewCloudControllerManagerServer() *CloudControllerManagerServer {
 
 // AddFlags adds flags for a specific ExternalCMServer to the specified FlagSet
 func (s *CloudControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
-	fs.Int32Var(&s.Port, "port", s.Port, "The port that the cloud-controller-manager's http service runs on")
-	fs.Var(componentconfig.IPVar{Val: &s.Address}, "address", "The IP address to serve on (set to 0.0.0.0 for all interfaces)")
+	fs.Int32Var(&s.Port, "port", s.Port, "The port that the cloud-controller-manager's http service runs on.")
+	fs.Var(componentconfig.IPVar{Val: &s.Address}, "address", "The IP address to serve on (set to 0.0.0.0 for all interfaces).")
 	fs.StringVar(&s.CloudProvider, "cloud-provider", s.CloudProvider, "The provider of cloud services. Cannot be empty.")
 	fs.StringVar(&s.CloudConfigFile, "cloud-config", s.CloudConfigFile, "The path to the cloud provider configuration file.  Empty string for no configuration file.")
```
Comment (xiangpengzhao) [inline cmd/cloud-controller-manager/app/options/options.go]: extra white space

File: cmd/cloud-controller-manager/app/options/options.go [hunk 2]
```diff
@@ -67,13 +67,13 @@ func NewCloudControllerManagerServer() *CloudControllerManagerServer {
...
 	fs.BoolVar(&s.AllowUntaggedCloud, "allow-untagged-cloud", false, "Allow the cluster to run without the cluster-id on cloud instances.  This is a legacy mode of operation and a cluster-id will be required in the future.")
```
Comment (xiangpengzhao) [inline cmd/cloud-controller-manager/app/options/options.go]: extra white space

File: cmd/cloud-controller-manager/app/options/options.go [hunk 3]
```diff
@@ -67,13 +67,13 @@ func NewCloudControllerManagerServer() *CloudControllerManagerServer {
...
-	fs.MarkDeprecated("allow-untagged-cloud", "This flag is deprecated and will be removed in a future release.  A cluster-id will be required on cloud instances")
-	fs.DurationVar(&s.MinResyncPeriod.Duration, "min-resync-period", s.MinResyncPeriod.Duration, "The resync period in reflectors will be random between MinResyncPeriod and 2*MinResyncPeriod")
+	fs.MarkDeprecated("allow-untagged-cloud", "This flag is deprecated and will be removed in a future release.  A cluster-id will be required on cloud instances.")
```
Comment (xiangpengzhao) [inline cmd/cloud-controller-manager/app/options/options.go]: extra white space

File: cmd/cloud-controller-manager/app/options/options.go [hunk 4]
```diff
@@ -82,14 +82,14 @@ func (s *CloudControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.RouteReconciliationPeriod.Duration, "route-reconciliation-period", s.RouteReconciliationPeriod.Duration, "The period for reconciling routes created for Nodes by cloud provider.")
 	fs.BoolVar(&s.ConfigureCloudRoutes, "configure-cloud-routes", true, "Should CIDRs allocated by allocate-node-cidrs be configured on the cloud provider.")
 	fs.BoolVar(&s.EnableProfiling, "profiling", true, "Enable profiling via web interface host:port/debug/pprof/")
```
Comment (xiangpengzhao) [inline cmd/cloud-controller-manager/app/options/options.go]: add period

File: cmd/cloud-controller-manager/app/options/options.go [hunk 5]
```diff
@@ -82,14 +82,14 @@ func (s *CloudControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
...
-	fs.BoolVar(&s.EnableContentionProfiling, "contention-profiling", false, "Enable lock contention profiling, if profiling is enabled")
+	fs.BoolVar(&s.EnableContentionProfiling, "contention-profiling", false, "Enable lock contention profiling, if profiling is enabled.")
 	fs.StringVar(&s.ClusterCIDR, "cluster-cidr", s.ClusterCIDR, "CIDR Range for Pods in cluster.")
 	fs.BoolVar(&s.AllocateNodeCIDRs, "allocate-node-cidrs", false, "Should CIDRs for Pods be allocated and set on the cloud provider.")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
```
Comment (xiangpengzhao) [inline cmd/cloud-controller-manager/app/options/options.go]: add period

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]
since you're here, can you please also fix other nits?

File: cmd/cloud-controller-manager/app/options/options.go (hunk 1 above)
Comment (drinktee) [reply cmd/cloud-controller-manager/app/options/options.go]: where?

Comment (drinktee) [reply cmd/cloud-controller-manager/app/options/options.go]: Done

**Comment by xiangpengzhao** [comment]
/release-note-none
/lgtm

Thanks for contribution!

**Comment by wlan0** [comment]
/lgtm

Thanks for the PR!

**Pull request merged**